tests/tacacs/utils.py

```diff
@@ -49,6 +52,8 @@ def setup_tacacs_server(ptfhost, creds_all_duts, duthost):
                   'tacacs_rw_user_passwd': crypt.crypt(creds_all_duts[duthost]['tacacs_rw_user_passwd'], 'abc'),
                   'tacacs_ro_user': creds_all_duts[duthost]['tacacs_ro_user'],
                   'tacacs_ro_user_passwd': crypt.crypt(creds_all_duts[duthost]['tacacs_ro_user_passwd'], 'abc'),
+                  'tacacs_authorization_user': creds_all_duts[duthost]['tacacs_authorization_user'],
+                  'tacacs_authorization_user_passwd': crypt.crypt(creds_all_duts[duthost]['tacacs_authorization_user_passwd'], 'abc'),
```

Comment:
>abc [](http://example.com/codeflow?start=128&length=3)

Could you explain what is `abc` ?

Reply:
According to python document: https://docs.python.org/3/library/crypt.html
crypt.crypt(word, salt=None)
The 'abc' is salt to help protect password hash result.